### PR TITLE
improvements to github exporter docs

### PR DESCRIPTION
**Pull request by danielsinai**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.

## Added docs pages

Please also include the path for the added docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- ...

## Updated docs pages

Please also include the path for the updated docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- ...

### CODE REVIEW

File: docs/exporters/github-exporter/exporting-all-repositories.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+---
+sidebar_position: 3
```
Comment (MPTG94) [inline docs/exporters/github-exporter/exporting-all-repositories.md:2]: Again, verify sidebar position numbering

**Review by MPTG94 — APPROVED** [review]
LGTM

File: docs/exporters/bitbucket-exporter/exporting-all-repositories.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+:::
+
+# Exporting all repositories
+
+By the end of this tutorial, we will have a `microservice` Blueprint that contains the properties `repository URL`, `Workspace` and `Project` which are automatically synced from Bitbucket to Port entity.
+
+1. Create a `microservice` Blueprint and `port-app-config.yml` configuration file.
+
+To export your Bitbucket `repositories` to Port, you can use the following Port Blueprints definitions, and `port-app-config.yml`:
```
Comment (EllaFurman) [inline docs/exporters/bitbucket-exporter/exporting-all-repositories.md:18]:
```suggestion
In this tutorial you will learn how to create a `microservice` Blueprint that contains the properties `repository URL`, `Workspace` and `Project`, which are automatically synced from Bitbucket to Port Entity.

1. Create a `microservice` Blueprint and `port-app-config.yml` configuration file.

To export your Bitbucket `repositories` to Port, you can use the following Port Blueprints definitions, and `port-app-config.yml`:
```

File: docs/exporters/bitbucket-exporter/exporting-all-repositories.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  },
+  "mirrorProperties": {},
+  "calculationProperties": {},
+  "relations": {}
+}
+```
+
+</details>
+
+You have to place the `port-app-config.yml` in the repository's root folder.
```
Comment (EllaFurman) [inline docs/exporters/bitbucket-exporter/exporting-all-repositories.md:54]:
```suggestion
Place the `port-app-config.yml` in the repository's root folder.
```

File: docs/exporters/bitbucket-exporter/exporting-files.md [hunk 4]
```diff
@@ -9,19 +9,13 @@ title: Exporting repositories
 
 :::
 
-# Exporting repositories
+# Exporting files
 
-By the end of this tutorial, we will have a `microservice` Blueprint that contains an auto-synced `REAMDE.md` file and `repository URL` from Bitbucket to Port properties.
+By the end of this tutorial, we will have a `microservice` Blueprint that contains an auto-synced README.md file into a Port property.
```
Comment (EllaFurman) [inline docs/exporters/bitbucket-exporter/exporting-files.md:14]:
```suggestion
In this tutorial you will learn how to create a `microservice` Blueprint that contains an auto-synced README.md file into a Port property.
```

File: docs/exporters/github-exporter/exporting-all-repositories.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+:::note Prerequisites
+
+- [Please install our GitHub app](./installation).
+
+:::
+
+# Exporting all repositories
+
+By the end of this tutorial, we will have a `microservice` Blueprint that contains the properties `repository URL` and `Description` which are automatically synced from GitHub to Port entity.
```
Comment (EllaFurman) [inline docs/exporters/github-exporter/exporting-all-repositories.md:14]:
```suggestion
IN this tutorial you will learn how to create a `microservice` Blueprint that contains the properties `repository URL` and `Description`, which are automatically synced from GitHub to Port Entity.
```

File: docs/exporters/github-exporter/exporting-all-repositories.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+  },
+  "mirrorProperties": {},
+  "calculationProperties": {},
+  "relations": {}
+}
+```
+
+</details>
+
+In order to apply the `port-app-config.yml` in the whole organization, place it in the `.github` folder or in the `.github-private` repository in the root directory.
```
Comment (EllaFurman) [inline docs/exporters/github-exporter/exporting-all-repositories.md:51]:
```suggestion
To apply the `port-app-config.yml` in the whole organization, place it in the `.github` folder or in the `.github-private` repository in the root directory.
```

File: docs/exporters/github-exporter/exporting-files.md [hunk 7]
```diff
@@ -9,9 +9,9 @@ title: Exporting repositories
 
 :::
 
-# Exporting repositories
+# Exporting files
 
-By the end of this tutorial, we will have a `microservice` Blueprint that contains an auto-synced `REAMDE.md` file and `repository URL` from GitHub to Port properties.
+By the end of this tutorial, we will have a `microservice` Blueprint that contains an auto-synced `README.md` file into a Port property.
```
Comment (EllaFurman) [inline docs/exporters/github-exporter/exporting-files.md:14]:
```suggestion
In this tutorial you will learn how to create a `microservice` Blueprint that contains an auto-synced `README.md` file into a Port property.
```

File: docs/exporters/github-exporter/self-hosted-installation.md [hunk 8]
```diff
@@ -44,7 +44,7 @@ In organizations that have a self-hosted GitHub installation there is no access
   - Metadata: Readonly
   - Issues: Readonly
   - Pull Request: Read and Write
-- Repository Events:
+- Repository Events (Needed to receive webhooks of changes from GitHub):
```
Comment (EllaFurman) [inline docs/exporters/github-exporter/self-hosted-installation.md:47]:
```suggestion
- Repository Events (required to receive webhook changes from GitHub):
```

File: docs/software-catalog/dashboards/dashboards.md [hunk 9]
```diff
@@ -30,11 +30,29 @@ You can create a pie chart illustrating data from Entities in your software cata
...
+| `Title`                 | `String` | The title of the visualization                                                                                                   | `null`  | `true`   |
+| `Icon`                  | `String` | The icon of the visualization                                                                                                    | `null`  | `false`  |
+| `Description`           | `String` | A short description to describe the visualization                                                                                | `null`  | `false`  |
+| `Blueprint`             | `String` | The Blueprint to visualize the data of its Related Entities                                                                      | `null`  | `true`   |
+| `Breakdown by property` | `String` | Group your chart by a specific property                                                                                          | `null`  | `true`   |
+| `Filters`               | `Array`  | Filters to include or exclue specific data based on Port's [Search Rules](../../software-catalog/search-in-port.md#search-rules) | []      | `false`  |
```
Comment (EllaFurman) [inline docs/software-catalog/dashboards/dashboards.md:40]:
```suggestion
| `Title`                 | `String` | Visualization title                                                                                                   | `null`  | `true`   |
| `Icon`                  | `String` | Visualization Icon                                                                                                     | `null`  | `false`  |
| `Description`           | `String` | Visualization description                                                                             | `null`  | `false`  |
| `Blueprint`             | `String` | The chosen Blueprint from which related Entities data is visualized from                                                                    | `null`  | `true`   |
| `Breakdown by property` | `String` | Group your chart by a specific property                                                                                          | `null`  | `true`   |
| `Filters`               | `Array`  | Filters to include or exclude specific data based on Port's [Search Rules](../../software-catalog/search-in-port.md#search-rules) | []      | `false`  |
```

File: docs/software-catalog/dashboards/dashboards.md [hunk 10]
```diff
@@ -30,11 +30,29 @@ You can create a pie chart illustrating data from Entities in your software cata
...
+
+#### Example: Filter only `Deployment` entities from last week
+
+Assuming we have a [Blueprint](../../software-catalog/blueprint/blueprint.md) that is called `Service` which is related to another called `Deployment` and we want to create visualizations on top of last week's deployments of this service.
+
+To achieve this desired state, we can go into one of the `Service`'s specific page and create a new visualization and after selecting the `Deployment` Blueprint in the dropdown we can add the following filter to the the `Filters` array:
```
Comment (EllaFurman) [inline docs/software-catalog/dashboards/dashboards.md:46]:
```suggestion
#### Example: filter only `Deployment` Entities from last week

Let's assume we have a [Blueprint](../../software-catalog/blueprint/blueprint.md) that is called `Service` which is related to another Blueprint called `Deployment`, and we want to create visualizations on top of the last week's deployments of this service.

To achieve this desired state, we can go into one of the `Service`'s profile pages and create a new visualization. After selecting the `Deployment` Blueprint in the dropdown, we can add the following filter to the `Filters` array:
```

**Review by EllaFurman — CHANGES_REQUESTED** [review]

**Pull request merged**